go.mod

```
@@ -8,7 +8,8 @@ require (
 	contrib.go.opencensus.io/exporter/prometheus v0.2.0
 	github.com/Azure/azure-pipeline-go v0.2.3 // indirect
 	github.com/Azure/azure-storage-blob-go v0.10.0
-	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
+	github.com/alecthomas/kingpin v0.0.0-20200323085623-b6657d9477a6
```

Do we need a comment here indicating that this is pulling `master`, which is newer `.v2`?
```suggestion
	// kingpin does not follow Go Modules versioning mechanism for versions > 1.
	// This is pulling from master which is newer than > v2.2.6
	github.com/alecthomas/kingpin v0.0.0-20200323085623-b6657d9477a6
```